boards/esp32s3-pros3/Makefile.features

```diff
@@ -0,0 +1,19 @@
+# the board uses a ESP32-S3 with a 16MB QSPI Flash and a 8MB QSPI PSRAM
+CPU_MODEL = esp32s3
+FEATURES_PROVIDED += esp_spi_ram
```

Comment:
This also needs to be provided in Kconfig

Reply:
Did I forget it ? :thinking:

Reply:
May I sqash it directly?

Reply:
Of course!

Reply:
Done